.github/labeler.yml

```diff
@@ -1,40 +1,36 @@
 Documentation:
 - changed-files:
-  - any-glob-to-any-file: docs/**
-
-NuGet:
-- changed-files:
-  - any-glob-to-any-file: '**/*.nuspec'
+  - any-glob-to-any-file: docs/** # Any modifications to Documentation
+  - any-glob-to-any-file: EXILED/Localization/** # Any modifications to Localization
 
-Scripts: # Add the 'Scripts' label
+GitHub:
 - changed-files:
+  - any-glob-to-any-file: '**/*.nuspec' #NuGet
   - any-glob-to-any-file: '**/*.ps1' # Windows
   - any-glob-to-any-file: '**/*.sh' # Linux
+  - any-glob-to-any-file: .github/** # Any modifications to github related files
 
-regarding-events: # Add the 'Events' label
+Events: # Add the 'Events' label
 - changed-files:
   - any-glob-to-any-file: EXILED/Exiled.Events/** # Any Modifications to events
+  - all-globs-to-all-files: '!EXILED/Exiled.Events/Patches/**/*' # Exept modifications to transpiler files
```

Comment:
Not sure why you felt patch changes were not part of the event but okay.

Reply:
I didn't see this change, but while I think what Yamato is trying to is seperate transpires and events into isolated labels, the intent for the labeled was to help provide a quick way of understanding what parts of Exiled were actually being modified.